orchestration/values.schema.json

```
@@ -85,7 +85,17 @@
       },
       "required": ["stagingData", "jadeData"]
     },
+    "repo" : {
+      "type": "object",
+      "properties": {
+        "apiUrl": {"type":  "string"},
```

I would change this to not imply that the api prefix must be included:
```suggestion
        "url": {"type":  "string"},
```
You'll need to add the `/api/v1` prefix to the tasks above, which is ok. That'll leave each task open to switch to new versions of the API as they appear.

---

To what level of task granularity would this make sense? I'd think it'd be per endpoint, meaning that each task within the `ingest-file` template prepends a version itself. Is that what you're suggesting?

---

This should just be "https://datarepo.terra.bio" (or the equivalent for dev). That's the only thing that varies from environment to environment. The tasks know (or should know) the details about exactly which endpoint they want to interact with.